appinventor/components/src/com/google/appinventor/components/runtime/SoundRecorder.java

```diff
@@ -115,6 +116,20 @@ void stop() {
       recorder.reset();
       recorder.release();
     }
+
+    void pause() {
+      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
```

Comment:
So, this functionality actually requires SDK 24 to work (it's not available on lower versions). The solution we currently use is to just ignore and not call the new functionality on lower SDKs, though a different solution could be raising an `ErrorOcurred` event with a message like "_Pause/Resume is not available for this SDK_".